tls/s2n_client_hello.c

```diff
@@ -270,6 +269,16 @@ int s2n_process_client_hello(struct s2n_connection *conn)
         S2N_ERROR(S2N_ERR_PROTOCOL_VERSION_UNSUPPORTED);
     }
 
+    if (conn->config->quic_enabled) {
+        ENSURE_POSIX(conn->actual_protocol_version >= S2N_TLS13, S2N_ERR_PROTOCOL_VERSION_UNSUPPORTED);
```

Comment:
Is there a test for this case?

Reply:
Yes, it already existed: https://github.com/awslabs/s2n/blob/e6314dd844c50a90f3a489a177f1ff479bebe7b7/tests/unit/s2n_client_hello_test.c#L231-L285

It isn't a new check, I just moved it from line 264 to line 273 to keep all the quic error checking in one place.